packages/playwright-core/src/server/registry/browserFetcher.ts

```diff
@@ -24,6 +23,8 @@ import { existsAsync } from '../../utils/fileUtils';
 import { debugLogger } from '../../common/debugLogger';
 import { download } from './download';
 
+const extract: typeof import('../../../bundles/zip/node_modules/extract-zip') = require('../../zipBundle').extract;
```

Comment:
Why not `typeof import('extract-zip')` similar to `yazl`?

Reply:
extract-zip does not have a definitely-typed module, so i'd need a code devDependency in package.json